Open Judge System/Web/OJS.Web/Areas/Administration/Views/Shared/EditorTemplates/PositiveShortInteger.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@using System.Dynamic
+
+@model short?
+    
+@{
+    dynamic htmlAttributes = new ExpandoObject();
```

Comment:
Do we use Expando object anywhere else? I don't think we do, please keep the code consistent, if we use anonymous objects in other places do the same here. You can also just use standard c# variables while you're in the code part.

Reply:
The alternative for what I want to do here is to use a Dictionary<string, object> or just copy the anonymous object, but with one more property in it, neither one of both will be consistent, since I didn't see this logic elsewhere.

Reply:
It turns out we need a optional property ("disabled") in the html attributes, but it's impossible to create an annonymous object with varying number of fields.